hive_agent/chat/chat_manager.py

```diff
@@ -17,6 +16,9 @@ def __init__(self, llm, user_id: str, session_id: str):
     async def add_message(
         self, db_manager: DatabaseManager, role: str, content: str
     ):
+        if not isinstance(content, str):
```

Comment:
what's the motivation for this? the params take `content: str` so it'll always be string

Reply:
I had an error earlier, but now I removed it and it seems to be owrking fine; for some reason earlier content was AgentResponse, not a string